src/core/tests/runtime/interpreter/unit_test.manifest

```diff
@@ -75,8 +75,8 @@ INTERPRETER.onnx_if_inside_loop
 
 # Legacy tests with unsupported features from opset4 LSTM/GRU/RNN
 # Peepholes input unsupported
-onnx_model_lstm_fwd_with_clip_peepholes
-onnx_model_lstm_bdir_short_input_seq_peepholes
+#onnx_model_lstm_fwd_with_clip_peepholes
+#onnx_model_lstm_bdir_short_input_seq_peepholes
```

Comment:
If you enabled some tests please remove these tests from this list:
```suggestion
```

Reply:
Updated it! Thanks!